desktop/cmp/persistenceManager/Types.ts

```diff
@@ -18,11 +18,20 @@ export interface PersistenceView<T extends PlainObject = PlainObject> {
     value: T;
 }
 
-export interface TreeView {
-    itemType: string;
+export type PersistenceViewTree = {
     text: string;
-    items?: TreeView[];
-    selected?: boolean;
-    id?: number;
-    isFavorite?: boolean;
-}
+    selected: boolean;
+} & (
+    | {
+          type: 'directory';
+          items: PersistenceViewTree[];
+      }
+    | {
+          type: 'view';
+          id: number;
+          isFavorite: boolean;
+      }
+    | {
+          type: 'divider';
```

Comment:
Would remove `divider`. To me, that’s a view-level concern and not a concern of the model / data.

Reply:
Without 'divider' how would we know where to put in the menuDivider?